connector/src/main/resources/config.yml

```diff
@@ -151,4 +151,8 @@ enable-proxy-connections: false
 # 1400 is the default.
 # mtu: 1400
 
+# Whether to use direct server methods to retrieve information such as block states.
+# Turning this off for Spigot will stop NMS from being used but will have a performance impact.
+use-adapters: true
+
```

Comment:
Should we have an option for move fixes too?

Reply:
While I'm not opposed, I see no reason. There's not much reason people would want the move fixes off, and performance shouldn't be an issue. The reason that is an issue is in case there's a Spigot instance that breaks with them enabled.

Something that *could* be a config option is teleport ID confirmation flexibility.

Reply:
You mean the position and resend thresholds?

Reply:
Yes.